config/checkstyle_checks.xml

```diff
@@ -457,6 +457,11 @@
     <!-- Javadoc Comments -->
     <module name="AtclauseOrder"/>
     <module name="InvalidJavadocPosition"/>
+    <module name="JavadocBlockTagLocation">
+      <property name="tags" value="author, deprecated, exception, hidden, noinspection, param"/>
```

Comment:
Default  + `@noinspection`.

Reply:
+1

Reply:
please add it as 
```
<!-- defaults -->
<property name="tags" value="...."/>
<property name="tags" value="..."/>
<!- extension -->
<property name="tags" value="noinspection"/>
```
to clearly show what is default and what is not.
Please add this as example to xdoc to show user how to do this.

Reply:
done